src/api/auth/authz/auth.js

```diff
@@ -0,0 +1,130 @@
+import http from "k6/http";
+import { logResult } from "../../../common/dynamicScenarios/utils.js";
+import {buildDefaultParams, CONFIG} from "../../../common/envVars.js";
+import authConfig from "../url.js";
+
+export const AUTH_API_NAMES = {
+  registerClient: "auth/registerClient",
+  revokeClient: "auth/revokeClient",
+  deleteOrganizationOperator: "auth/deleteOrganizationOperator",
+  getClients: "auth/getClients",
+  getClientSecret: "auth/getClientSecret",
+  getOrganizationOperator: "auth/getOrganizationOperator",
+  getOrganizationOperators: "auth/getOrganizationOperators",
+  getUserInfoFromMappedExternaUserId: "auth/getUserInfoFromMappedExternaUserId"
+};
+
+const baseUrl = CONFIG.USE_INTERNAL_ACCESS_ENV
```

Comment:
move also the ternary operator on the baseUrl configuration exported on the authConfig file, instead of repeating this logic everywhere

Reply:
Modified